.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,19 @@
-**Changes proposed in this pull request**
--
--
--
+<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->
+Fixes #
 
-**Screenshots (if appropriate)** 
+#### Checklist
 
-**Link to live demo (if appropriate):** 
+- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added necessary documentation (if appropriate)
 
-**Closes #**
+#### Changes proposed in this pull request:
+
+<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->
+-
+
+<!-- Demo Link: Add here the link where you changes can be seen. -->
```

Comment:
Let's make Demo link a heading :)

Reply:
I think heading should be set to Fixes # or Addresses # only

Reply:
If it's atleast bold, it'd feel more compelling if left empty, so contributors will generate a demo link, otherwise as you wish.